content/en/docs/concepts/workloads/pods/ephemeral-containers.md

```diff
@@ -76,3 +76,195 @@ you can view processes in other containers.
+                                                                                                                                                                                                                                                                                                }
+
+
+Run the program and pass the namespace, pod name, and path to a kube config file. We assume that the ephemeral-demo Pod is still running.
+
+go run main.go default ephemeral-demo $KUBECONFIG
+
+If it completes with no error a privileged ephemeral container should have been added to the Pod. Exec into it and list the host's devices to prove that it is a privileged container.
+
+kubectl exec -it ephemeral-demo -c debug -- sh
+ls /dev
+
+### Conclusion
+
+If there is one takeaway from this post, it is that any policy tool that has not been updated in the last couple of months will not enforce rules on ephemeral containers. This also includes all policies written in house! It is not enough to update the community policies.
+
+Some may argue that this type of oversight is not really an issue. Ephemeral containers can't mount host paths, or access the hosts namespaces. All it can do is set the common container security context. That is a fair comment, because it's true. Being able to create a privileged container is however still not ideal, and there are methods to escalate privileges when this is possible. Either way it is important to be aware of how policies are enforced and the security contexts which are allowed.
+
+I am still not sure how much of an issue this will be short term. Cloud providers are currently in the process of rolling out Kubernetes v1.23 in their SaaS offerings. In these solutions it is still a possibility that they chose to disable ephemeral containers. Those rolling their own clusters may have already upgraded to v1.23 and not be aware of the new feature. That is the biggest issue really, that the platform administrator has to be aware of the existence of ephemeral containers. The fact that kubectl does not expose the option to set a security context will make even less people aware that it is still possible to set one with other means. Investing in a security audit 6 months ago will only be valuable as long as the same Kubernetes version is used. Kubernetes is by design not secure by default, so each new feature introduced has to be analyzed. The fact that upgrading from Kubernetes v1.22 to v.23 could make your cluster less secure is part of the difficulties of working with Kubernetes, requiring platform administrators to always stay on top of things. The reality is that these types of things are easy to miss, so hopefully this post has helped someone make their cluster a bit more secure.
+
```

Comment:
This is a good blog article if you want to publish it somewhere else.
However, the taste of this page is very different from what we host on the k8s.io/docs website.

Reply:
@tengqm @windsonsea  thanks for ur doc review it's my first documentation for an open source project so I didn't know the guidelines so I may made some mistakes and I  will follow ur suggestions for the  changes proposed